packages/core/core/src/public/BundleGraph.js

```diff
@@ -90,6 +90,15 @@ export default class BundleGraph<TBundle: IBundle>
       .map(dep => new Dependency(dep));
   }
 
+  getAssetWithDependency(dep: IDependency): ?IAsset {
```

Comment:
Can we remove `Dependency#sourceAssetId` now? This is essentially equal to `getAssetById(dep.sourceAssetId)`

Reply:
Ah, I hadn't thought to use that. Looks unused, yeah.